lib/qa_topic_edits.rb

```diff
@@ -61,10 +61,32 @@ class ::Topic
 
   def self.voted(topic, user)
```

Comment:
We want to get rid of this method altogether. And the associated property and method in the serializer.  We don't need it anymore.